guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md

```diff
@@ -148,8 +148,7 @@ A column can have the following attributes:
 </tr>
 </table>
 
-For more information about each type, refer to the annotations in the corresponding XSD file. The XSD files are located in the `<Magento_root_directory/setup/src/Magento/Setup/Model/Declaration/Schema/etc` directory.
-
+For more information about each type, refer to the annotations in the corresponding XSD file. The XSD files are located in the `<Magento_root_directory/setup/src/Magento/Setup/Model/Declaration/Schema/etc` or `<Magento_root_directory/lib/internal/Magento/Framework/Setup/Declaration/Schema/etc` directory. There are two possible places where the mentioned XSD files can be located. It depends on how Magento is installed.
```

Comment:
Please correlate each path with a specific installation method.

Reply:
@jeff-matthews I have updated, Can you please proceed this PR?